dir.props

```diff
@@ -18,6 +19,15 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <!-- Temporarily omit code analysis -->
+    <SkipImportCodeAnalysisTargets>true</SkipImportCodeAnalysisTargets>
+    <!-- Hacky BuildConfiguration -->
+    <BuildConfiguration Condition="'$(BuildConfiguration)' == ''">netcoreapp1.1-$(OSEnvironment)</BuildConfiguration>
+  </PropertyGroup>
+  
+  <Import Condition="Exists('$(MSBuildProjectDirectory)/Configurations.props')" Project="$(MSBuildProjectDirectory)/Configurations.props" />
```

Comment:
Is MSBuildProjectDirectory correct? I'm not sure where configuration.props is supposed to come from actually.